php.ini-development

```diff
@@ -890,83 +893,83 @@ default_socket_timeout = 60
 ;auto_detect_line_endings = Off
 
 ;;;;;;;;;;;;;;;;;;;;;;
-; Dynamic Extensions ;
+; dynamic extensions ;
 ;;;;;;;;;;;;;;;;;;;;;;
 
 ; If you wish to have an extension loaded automatically, use the following
 ; syntax:
 ;
-;   extension=modulename
+;   extension = modulename
 ;
 ; For example:
 ;
-;   extension=mysqli
+;   extension = "mysqli"
 ;
 ; When the extension library to load is not located in the default extension
 ; directory, You may specify an absolute path to the library file:
 ;
-;   extension=/path/to/extension/mysqli.so
+;   extension = "/path/to/extension/mysqli.so"
 ;
 ; Note : The syntax used in previous PHP versions ('extension=<ext>.so' and
 ; 'extension='php_<ext>.dll') is supported for legacy reasons and may be
 ; deprecated in a future PHP major version. So, when it is possible, please
-; move to the new ('extension=<ext>) syntax.
+; move to the new ('extension = <ext>) syntax.
 ;
 ; Notes for Windows environments :
 ;
 ; - Many DLL files are located in the ext/
 ;   extension folders as well as the separate PECL DLL download.
 ;   Be sure to appropriately set the extension_dir directive.
 ;
-;extension=bz2
-;extension=curl
-;extension=exif
-;extension=ffi
-;extension=ftp
-;extension=fileinfo
-;extension=gd
-;extension=gettext
-;extension=gmp
-;extension=intl
-;extension=ldap
-;extension=mbstring
-;extension=mysqli
-;extension=odbc
-;extension=openssl
-;extension=pdo_firebird
-;extension=pdo_mysql
-;extension=pdo_odbc
-;extension=pdo_pgsql
-;extension=pdo_sqlite
-;extension=pgsql
-;extension=shmop
+;extension = "bz2"
+;extension = "curl"
+;extension = "exif"
+;extension = "ffi"
+;extension = "ftp"
+;extension = "fileinfo"
+;extension = "gd"
+;extension = "gettext"
+;extension = "gmp"
+;extension = "intl"
+;extension = "ldap"
+;extension = "mbstring"
+;extension = "mysqli"
+;extension = "odbc"
+;extension = "openssl"
+;extension = "pdo_firebird"
+;extension = "pdo_mysql"
+;extension = "pdo_odbc"
+;extension = "pdo_pgsql"
+;extension = "pdo_sqlite"
+;extension = "pgsql"
+;extension = "shmop"
 
 ; The MIBS data available in the PHP distribution must be installed.
 ; See https://www.php.net/manual/en/snmp.installation.php
-;extension=snmp
+;extension = "snmp"
 
-;extension=soap
-;extension=sockets
-;extension=sodium
-;extension=sqlite3
-;extension=tidy
-;extension=xsl
-;extension=zip
+;extension = "soap"
+;extension = "sockets"
+;extension = "sodium"
+;extension = "sqlite3"
+;extension = "tidy"
+;extension = "xsl"
+;extension = "zip"
 
-;zend_extension=opcache
+;zend_extension = "opcache"
 
 ;;;;;;;;;;;;;;;;;;;
-; Module Settings ;
+; module settings ;
 ;;;;;;;;;;;;;;;;;;;
 
-[CLI Server]
+[cli server]
```

Comment:
Ditto, these are "section" titles

Reply:
Again, in the current master we've got:

[intl]
[filter]
[mail function] – all lowercase
[Session] – first letter uppercase
[ODBC] – all caps
[MySQLi] but [mysqlnd]
[Pcre]
[bcmath]

Isn't it much simpler to go with lowercase all the way?

Reply:
These are extension names so these are proper nouns. Having said that, some of them look incorrect. 

- https://www.pcre.org/ should be PCRE
- https://www.php.net/manual/en/book.bc.php & https://github.com/php/php-src/blob/master/ext/bcmath/CREDITS should be BC Math
- filter should probably be Filter https://www.php.net/manual/en/book.filter.php